xml/ay_bigfile.xml

```diff
@@ -17748,16 +17765,17 @@ mv /var/run/zypp.sav /var/run/zypp.pid
    <para>
     Even if parameters like <literal>hostip</literal>,
     <literal>nameserver</literal>, and <literal>gateway</literal> are passed to
-    linuxrc, the network is only started when it is needed (for example, when
-    installing via SSH or VNC). Since <literal>autoyast</literal> is not a linuxrc
-    parameter (this parameter is ignored by linuxrc and only passed to &yast;),
-    the network will <emphasis>not</emphasis> be started automatically when
-    specifying a remote location for the &ay; profile.
+    <command>linuxrc</command>, the network is only started when it is needed
+    (for example, when installing via SSH or VNC). Since <literal>autoyast</literal>
```

Comment:
replace 'since' with 'because' or 'as'? ('since' can have multiple meanings, but you want the causal meaning here)